practices/practice-one/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,21 @@
+import { ChangeEvent } from "react";
+import "./input.css";
+
+type CustomStyle = {
+  [key: string]: string;
+};
+
+interface InputProps {
```

Comment:
it's better to inherit props from InputHTMLAttributes

Reply:
I added it